src/toil/leader.py

```diff
@@ -159,6 +160,8 @@ def __init__(self,
             # Message bus messages need to go to the given file.
             # Keep a reference to the return value so the listener stays alive.
             self._message_subscription = self.toilState.bus.connect_output_file(self.config.write_messages)
+        else:
+            raise RuntimeError("Write messages not set")
```

Comment:
Since `write_messages` is now type `str`, it isn't allowed by the type system to be None, so we probably don't actually need to check if it is None. We can just use it as a string.

The case we *might* have to handle is if it is a file path that can't be opened (like `""`). But `connect_output_file` will probably take care of throwing an error in that case.